SAG-Mill.md

```diff
@@ -1,2 +1,26 @@
-![](http://loenwind.info/eio/SAG_Mill.png
-)
\ No newline at end of file
+![](http://loenwind.info/eio/SAG_Mill.png)
+
+Crushes items.  Used for ore doubling and as a crafting process.
+
+The slot on the right side of the GUI is for SAG Mill Grinding Balls.  These additives increase the yield and reduce the power required for certain recipes.  The Grinding Balls are consumed, but do last for several operations.  The Main Output is a bonus to all of the possible outcomes.  The Bonus Output is applied multiplicatively to any secondary outputs.  See the example at the bottom of the page.
+
+| Grinding Ball    | Main Output | Bonus Output | Power Reduction |
+| :--------------- | ----------: | -----------: | --------------: |
+| Flint            |        120% |         125% |             15% |
+| Dark Steel Balls |        150% |         200% |             30% |
+
+Upgrade with capacitors to increase speed and energy storage.
+
+| Capacitor           | RF/t | Buffer (RF) |
+| :------------------ | ---: | ----------: |
+| Basic (none)        |   20 |     100,000 |
+| Double-Layer        |   60 |     300,000 |
+| Octadic             |  100 |     500,000 |
+| Good Loot Cap (4)   |  160 |     800,000 |
+| Theoretical Max (5) |  200 |   1,000,000 |
+
+***
+
+Without a Grinding Ball, Diamond Ore will give you 1 diamond with a 25% chance of an additional diamond, or an average of 1.25 diamonds per ore.
+
+Adding a Dark Steel Ball will give you  a 150% chance of the main diamond, or 1.5 diamonds per ore..  The secondary diamond ends up being 25% * 150% * 200% = 75%, or 0.75 diamonds per ore.  Combining this with the primary diamond chance, a SAG Mill with Dark Steel Balls will, on average, get 2 diamonds per ore.
```

Comment:
1.5 + 0.75 = 2?

Reply:
What's wrong with that?  Are you now going to tell me that 2 + 2 =/= 5? </s>.